src/sdk-reference/js/6/profile/introduction/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: sdk.html.hbs
+title: Introduction
+description: Profile class
+order: 0
+---
+
+# Profile
+
+This class represents a Kuzzle Profile.  
+
+Refers to the [Security guide]({{ site_base_path }}guide/1/essentials/security#defining-profiles-default) for more informations about profiles.
+
+## Properties
+
+Available properties.
+
+| Property | Type | Description | Writable? |
+|--- |--- |--- | :-------: |
+| `_id` | <pre>string</pre> | Profile ID |    No     |
```

Comment:
The `Writable?` column has to go: these properties are not defined as read-only (and I don't see why they should be)